apps/src/javalab/PreviewPaneHeader.jsx

```diff
@@ -42,7 +42,11 @@ export default function PreviewPaneHeader({
       {showAssetManagerButton && (
         <PaneButton
           headerHasFocus
-          onClick={() => assets.showAssetManager()}
+          onClick={() =>
+            assets.showAssetManager(null, null, null, {
+              customAllowedExtensions: '.wav, .jpg, .jpeg, .jfif, .gif, .png'
```

Comment:
Do we support gif for images?

Reply:
I just tried to put a gif in and it threw an exception, so if we don't need to support it on the curriculum side I vote we don't try to fix it pre-pilot.

Reply:
Sounds good, I'll remove it from here for now.